WBooks/src/app/components/IconHeader/styles.js

```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  image: {
+    height: 20,
```

Comment:
You are using several times height and width `20` for images, why don't you abstract that?